.github/CODEOWNERS

```diff
@@ -27,6 +27,11 @@
 # PRLabel: %OpenAI
 /sdk/ai                       @richardpark-msft @jhendrixMSFT @chlowell
 
+# AzureSDKOwners: @richardpark-msft
+# ServiceLabel: %OpenAI
+# PRLabel: %OpenAI
+/sdk/ai/azopenai                       @richardpark-msft @jhendrixMSFT @chlowell @achauhan-scc @PratibhaShrivastav18 @kingernupur @glecaros
```

Comment:
Please drop Joel and Charles from the code owner list for this folder. Its ok if you are on the code owner list.

Reply:
Done!